### PR TITLE
Docker ce update

### PR DESCRIPTION
**Pull request by jmarcet**
Maintainer: Gerard Ryan <G.M0N3Y.2503@gmail.com>
Compile tested: master x86-64
Run tested: master x86-64

Description:
After updating my PR #11363, #11364 & #11365 for docker-compose, I realized we had several docker related packages which were lagging behind current versions.

This PR updates containerd to v1.3.4, tini to v0.19.0, libnetwork to commit 28277c0c from June 2nd and docker-ce to v19.03.11

I'm still testing it, but so far this seems to really improve the nasty netns race condition I've been facing (and chasing in the kernel) for months. I'll know more in the coming days.

### CODE REVIEW

**Comment by neheb** [comment]
ping @G-M0N3Y-2503

File: utils/docker-ce/Makefile [hunk 1]
```diff
@@ -22,10 +22,10 @@ define CheckExpectedSrcVer
 endef
 
 # values from respective '.installer' files at https://github.com/docker/docker-ce/blob/v$(PKG_VERSION)/components/engine/hack/dockerfile/install/
-$(eval $(call CheckExpectedSrcVer,../containerd/Makefile,7ad184331fa3e55e52b890ea95e65ba581ae3429))
-$(eval $(call CheckExpectedSrcVer,../libnetwork/Makefile,0941c3f409260d5f05cfa6fc68420d8ad45ee483))
+$(eval $(call CheckExpectedSrcVer,../containerd/Makefile,814b7956fafc7a0980ea07e950f983d0837e5578))
+$(eval $(call CheckExpectedSrcVer,../libnetwork/Makefile,28277c0c98d0dba7a361d3d448975c4237625f37))
 $(eval $(call CheckExpectedSrcVer,../runc/Makefile,dc9208a3303feef5b3839f4323d9beb36df0a9dd))
```
Comment (G-M0N3Y-2503) [inline utils/docker-ce/Makefile:27]: [`docker-ce` v19.03.11 uses `runc` v1.0.0-rc10](https://github.com/docker/docker-ce/blob/v19.03.11/components/engine/hack/dockerfile/install/runc.installer#L7).
I don't think anything else currently depends on `runc` so we should probably just leave it with what the version of `docker-ce` wants.
Also, if I'm not mistaken, this version bump is one of the major changes in v19.03.11.

**Comment by jmarcet** [comment]
OK, I see. I was excited thinking this could fix the [race condition](https://bugzilla.kernel.org/show_bug.cgi?id=207403) I mentioned, but it does not, the race is still there.

Thus, following your explanation I'm closing the PR. Before doing so, are you aware of that bug? I would love to have it fixed.

File: utils/docker-ce/Makefile (hunk 1 above)
Comment (jmarcet) [reply utils/docker-ce/Makefile:27]: Ditto.

File: utils/docker-ce/Makefile [hunk 2]
```diff
@@ -22,10 +22,10 @@ define CheckExpectedSrcVer
 endef
 
 # values from respective '.installer' files at https://github.com/docker/docker-ce/blob/v$(PKG_VERSION)/components/engine/hack/dockerfile/install/
-$(eval $(call CheckExpectedSrcVer,../containerd/Makefile,7ad184331fa3e55e52b890ea95e65ba581ae3429))
-$(eval $(call CheckExpectedSrcVer,../libnetwork/Makefile,0941c3f409260d5f05cfa6fc68420d8ad45ee483))
+$(eval $(call CheckExpectedSrcVer,../containerd/Makefile,814b7956fafc7a0980ea07e950f983d0837e5578))
+$(eval $(call CheckExpectedSrcVer,../libnetwork/Makefile,28277c0c98d0dba7a361d3d448975c4237625f37))
```
[the remark this answers is not in the document]
Comment (jmarcet) [reply utils/docker-ce/Makefile:26]: Ditto

**Comment by G-M0N3Y-2503** [comment]
I was not aware, but an interesting find nevertheless.

**Comment by jmarcet** [comment]
All right, I close the PR.

**Pull request closed without merging**